docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
What if we drop any kind of validation in the "new-sourcecfg" so user could choose from the set of fields (username/ pw/ cert/ sshkey/ gitconfig) and define only those he really needs for accessing the source and dont split them at all ?